tools/remote_build/rbe_common.bazelrc

```diff
@@ -53,15 +58,15 @@ build --test_timeout=300,450,1200,3600
 build:asan --copt=-gmlt
 # TODO(jtattermusch): use more reasonable test timeout
 build:asan --test_timeout=3600
-build:asan --test_tag_filters=-qps_json_driver
+build:asan --test_tag_filters=-no_linux,-qps_json_driver
 
```

Comment:
Are sanitizers only available on Linux?

Reply:
Good question. This is "rbe_common.bazelrc" file so it only has configuration that's used for bazel RBE builds and that config is only used for linux RBE builds (windows has its own .rc file). So in this particular case (Bazel RBE), the answer is yes, sanitizers only run on linux.